services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java

```diff
@@ -86,16 +86,8 @@ public HealthCheck getHealthCheck() {
 
     @Override
     public FlowPayload getFlow(String flowId) {
-        try {
-            return restTemplate.exchange("/api/v1/flows/{flow_id}", HttpMethod.GET,
-                    new HttpEntity(buildHeadersWithCorrelationId()), FlowPayload.class, flowId).getBody();
-        } catch (HttpClientErrorException ex) {
-            if (ex.getStatusCode() != HttpStatus.NOT_FOUND) {
-                throw ex;
-            }
-
-            return null;
-        }
+        return restTemplate.exchange("/api/v1/flows/{flow_id}", HttpMethod.GET,
```

Comment:
this change may affect atdd-staging

Reply:
Yes, it definitely will, but atdd-staging is already broken now. We will fix it when fixing atdd-staging.